cluster.yml

```diff
@@ -10,6 +10,9 @@ auth:
 # calico
 # canal
 # weave
+#
+# If you are using calico on AWS, use the network plugin config option:
+# 'aws: true'
```

Comment:
@moelsayed as a second thought, got concerned if there may be other cloud providers requiring special handling for calico plugins? Or is it just aws. If there is more than one cloud provider, then may be we should consider changing it to `cloudprovider: aws/gce/etc`. Let me know